packages/terra-base/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ Unreleased
 5.34.0 - (June 9, 2020)
 ------------------
 ### Changed
+* Directly include packaged themes.
```

Comment:
shouldn't be this in Unreleased section 🙄

Reply:
updated here https://github.com/cerner/terra-core/pull/3015/commits/f11c6a73661a88a9e3db110ffb03e848b93b4fd4